src/main/cpp/Utils.cpp

```diff
@@ -33,15 +33,12 @@ namespace TeamOKC {
         return degrees * M_PI / 180.0;
     }
     double sign(double value) {
-        if (value>0) {
+        if (value > 0) {
             return 1;
-        }
-        if (value<0) {
+        } else if (value < 0) {
             return -1;
         }
-        if (value==0) {
-            return 0;
-        }
+        return 0;
```

Comment:
This is fine, and I am not requesting changes. However, for your edification, you can also use a ternary operator like this:
```
double val = value < 0 ? -1.0 : 1.0;
```

There's no reason to `return 0` for `value == 0.0` because `1 * 0 = 0`. But if you insist on doing this you could nest the ternary operator:
```
double val = value < 0.0 ? -1.0 : (value > 0.0 ? 1.0 : 0.0);
```
and the parentheses aren't required in that last example (but it is much more readable)